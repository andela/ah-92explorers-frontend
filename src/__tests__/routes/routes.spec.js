import React from 'react';
import { shallow } from 'enzyme';
import Router from '../../components/Routes';

describe('<Router />', () => {
  it('should render routes', () => {
    const component = shallow(<Router />);
    expect(component).toMatchSnapshot();
  });
});
