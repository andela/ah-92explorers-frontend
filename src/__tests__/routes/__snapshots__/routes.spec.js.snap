// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Router /> should render routes 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Home)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Signup)",
          "type": [Function],
        }
      }
      path="/signup"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Continue)",
          "type": [Function],
        }
      }
      path="/continue"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(LoginContainer)",
          "type": [Function],
        }
      }
      exact={true}
      path="/login"
    />
  </Switch>
</BrowserRouter>
`;
